universal-application-tool-0.0.1/app/views/LoginForm.java

```diff
@@ -0,0 +1,32 @@
+package views;
+
+import static j2html.TagCreator.body;
+import static j2html.TagCreator.form;
+import static j2html.TagCreator.h1;
+
+import javax.inject.Inject;
+import play.mvc.Http;
+import play.twirl.api.Content;
+
+public class LoginForm extends BaseHtmlView {
+
+  private ViewUtils viewUtils;
+
+  @Inject
+  public LoginForm(ViewUtils viewUtils) {
+    this.viewUtils = viewUtils;
+  }
```

Comment:
suggestion (blocking): since you're not using `viewUtils` you can remove it from the class

Reply:
Done.